src/chrome/content/rules/Etiqa.com.my.xml

```diff
@@ -0,0 +1,65 @@
+<!--
+	For other Maybank coverage, see Maybank.com.xml
+
+
+	Non-functional subdomains:
+
+		- (www.)etiqa.com.my	(h)
+		- bpp	(e)
+		- dev.bpp	(t)
+		- cgs	(r)
+		- cls	(r)
+		- clsgl	(r)
+		- d	(m)
+		- dcatalyst	(m)
+		- edmtrace	(m)
+		- insc1	(t)
+		- insc2	(t)
+		- intranet	(r)
+		- lpappuat1	(t)
+		- partner	(e)
+		- pentaagent	(t)
+	
+	e: expired certificate
+	h: http redirect
+	i: invalid certificate chain
+	m: certificate mismatch
+	r: connection refused
+	s: self-signed certificate
+	t: timeout on https
+-->
+<ruleset name="Etiqa Insurance">
+
+	<!-- only MyAccount & getonline directories are functional -->
+	<target host="etiqa.com.my" />
+	<target host="www.etiqa.com.my" />
+	<target host="academy.etiqa.com.my" />
+	<target host="agentmail.etiqa.com.my" />
+	<target host="media.etiqa.com.my" />
+	<target host="t.etiqa.com.my" />
+	<target host="w.etiqa.com.my" />
```

Comment:
Can you provide a test url for `t` and `w`? They don't send anything on  `$`.

Reply:
Removed targets